src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Twig/WysiwygExtension.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+/**
+ * Copyright 2019 SURFnet B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+namespace Surfnet\ServiceProviderDashboard\Infrastructure\DashboardBundle\Twig;
+
+use HTMLPurifier;
+use HTMLPurifier_Config;
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFilter;
+
+class WysiwygExtension extends AbstractExtension
+{
+    /**
+     * @var HTMLPurifier
+     */
+    private static $purifier;
+
+    public function __construct()
+    {
+        $config = HTMLPurifier_Config::createDefault();
+
+        $config->set('Core.Encoding', 'UTF-8'); // replace with your encoding
+        $config->set('HTML.Doctype', 'XHTML 1.0 Transitional'); // replace with your doctype
```

Comment:
Our doctype is set to `html` (5). That s(c)ould be reflected in this config?!

Reply:
Unfortunately not, http://htmlpurifier.org/live/configdoc/plain.html#HTML.Doctype

Reply:
I don't think it would do any damage either with the used subset of allowed html.

Reply:
Would not setting it be an option?

Reply:
And maybe remove the `// replace with your ..` comments?

Reply:
Removed the comments and used a specialized html5 htmlpurifier.